src/main/java/com/fasterxml/jackson/databind/ObjectMapper.java

```diff
@@ -3521,7 +3521,8 @@ public <T extends JsonNode> T valueToTree(Object fromValue)
 
         // inlined 'writeValue' with minor changes:
         // first: disable wrapping when writing
-        final SerializationConfig config = getSerializationConfig().without(SerializationFeature.WRAP_ROOT_VALUE);
+        // [databind#4047] ObjectMapper.valueToTree will ignore the configuration SerializationFeature.WRAP_ROOT_VALUE
```

Comment:
comment is slightly misleading: the issue is that it would ignore WRAP_ROOT_VALUE, but after this change it doesn't anymore. the comment makes it sound like now it's ignoring WRAP_ROOT_VALUE

Reply:
Yeah sorry; I just copied issue description which is misleading. If anyone can do a PR thatd be great (or I'll fix that when I get home).